integration/__tests__/lock_test.js

```diff
@@ -1,80 +0,0 @@
-const {
```

Comment:
This is very out-dated compared to the new scenarios that it provides little to no value.